WPMedia/fusion-news-theme-blocks
# Added overflow to section-title links

## Description

Section title links are not centered properly in the mobile view; having more than one line of the links means no gap between the links and the right edge.

https://dev.themesinternal.arcpublishing.com/pf/all-blocks-test-ta/?_website=the-gazette

## Jira Ticket

- [THEMES-1466](https://arcpublishing.atlassian.net/browse/THEMES-1466)

## Acceptance Criteria

Having more than one line of the links should be centered on all links.

## Test Steps

1. Checkout this branch `git checkout THEMES-1466`
2. Run fusion repo with linked blocks `npx fusion start -f -l @wpmedia/section-title-block`
3. Set content source to site-service-hierarchy
4. Inspect the links in mobile view to see all are aligned to center without overflow

## Effect Of Changes

### Before

<img width="1710" alt="Screenshot 2024-03-20 at 3 33 13 PM" src="https://github.com/WPMedia/arc-themes-blocks/assets/83021791/2eba8655-f6c1-49cc-8cac-d05ef28c9d73">

### After

<img width="1711" alt="Screenshot 2024-03-20 at 3 33 29 PM" src="https://github.com/WPMedia/arc-themes-blocks/assets/83021791/51ba36a3-289e-4e6e-9f79-b5505cebaa63">

## Dependencies or Side Effects

- Feature pack pr for local development: https://github.com/WPMedia/arc-themes-feature-pack/pull/515
- How to deploy to news: https://arcpublishing.atlassian.net/wiki/spaces/TI/pages/3138682964/News+Theme+2.0+Migration+Development+Deployment+Notes

## Author Checklist

_The author of the PR should fill out the following sections to ensure this PR is ready for review._

        please explain why (so that we can fix it whenever it gets refactored).

## Reviewer Checklist

_The reviewer of the PR should copy-paste this template into the review comments on review._



[THEMES-1466]: https://arcpublishing.atlassian.net/browse/THEMES-1466?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
